### PR TITLE
feat: add tlsConfig.caCert to greenhouse prom

### PR DESCRIPTION
**Pull request by IvoGoman**
## Description
This is required for Plutono to authenticate against the Prometheus

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

- Related Issue # (issue)
- Closes # (issue)
- Fixes # (issue)

> Remove if not applicable

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for tests # (issue/pr)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added to documentation?

- [ ] 📜 README.md
- [ ] 🤝 Documentation pages updated
- [ ] 🙅 no documentation needed
- [ ] (if applicable) generated OpenAPI docs for CRD changes

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: charts/greenhouse/templates/pluginconfig-kube-monitoring.yaml [hunk 1]
```diff
@@ -20,6 +28,11 @@ spec:
       cluster: {{ .Values.monitoring.extLabels.cluster }}
       region: {{ .Values.monitoring.extLabels.region }}
 {{- end }}
+  - name: kubeMonitoring.prometheus.tlsConfig.caCert
+    valueFrom:
+      secret:
+        key: "ca.crt"
+        name: kube-monitoring-ingress-ca
```
Comment (richardtief) [inline charts/greenhouse/templates/pluginconfig-kube-monitoring.yaml:35]: This option is no longer used. I neglected to remove this option from the readme file. 
```suggestion
  - name: kubeMonitoring.prometheus.annotations.nginx\.ingress\.kubernetes\.io/auth-tls-secret
    value: greenhouse/kube-monitoring-ingress-ca
```

**Review by richardtief — CHANGES_REQUESTED** [review]
We can use the annotation directly. 

https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#client-certificate-authentication

File: charts/greenhouse/templates/pluginconfig-kube-monitoring.yaml (hunk 1 above)
Comment (IvoGoman) [reply charts/greenhouse/templates/pluginconfig-kube-monitoring.yaml]: Thanks :) 
Added this as another example for our discussion w.r.t. to rendering hard to read `PluginOptions` on the PluginAdmin UI. Not only deeply nested but also requires the additional escaping.

**Review by richardtief — APPROVED** [review]
Thanks

**Pull request merged**